src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const logo = document.querySelector('.logo');
+const message = document.createElement('div');
+
+message.classList.add('message');
+message.innerText = 'Promise was resolved!';
+
+const messageError = document.createElement('div');
+
+messageError.classList.add('message');
+messageError.classList.add('error-message');
+messageError.innerText = 'Promise was rejected!';
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve();
+    // if (logo) {
+    //   resolve(document.querySelector('body').appendChild(message));
+    // } else {
+    //   reject(document.querySelector('body').appendChild(messageError));
+    // }
+  });
+});
+
+promise1
+  .then(() => {
+    body.appendChild(message);
+  })
+  .catch(() => {
+    body.appendChild(messageError);
+  });
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(document.querySelector('body').appendChild(messageError));
```

Comment:
The `reject()` function is called with an argument that appends the error message to the body. This operation should be handled in the `.catch()` block instead of within the promise executor function.